src/compose-create-pull-request.ts

```diff
@@ -79,13 +79,13 @@ export async function composeCreatePullRequest(
   // https://developer.github.com/v3/repos/commits/#list-commits-on-a-repository
   const {
     data: [latestCommit],
-  } = await octokit.request("GET /repos/:owner/:repo/commits", {
+  } = await octokit.request("GET /repos/{owner}/{repo}/commits", {
     owner,
     repo,
     sha: base,
     per_page: 1,
   });
-  state.latestCommitSha = latestCommit.sha;
+  state.latestCommitSha = latestCommit.sha as string;
```

Comment:
Can you update `package-lock.json` so that `@octokit/types` 6.3.0+ is used? Then you can remove the ` as string`

```suggestion
  state.latestCommitSha = latestCommit.sha;
```